content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -109,3 +109,26 @@ spec:
       weight: 10
 EOF
 {{< /text >}}
+
+### Authorization policy attachment
+
+In a scenario where policy requires application layer attributes, such as HTTP verbs, a waypoint proxy is required. It is important to understand that ztunnel cannot meaningfully enforce any policy that requires L7 parsing and so if they are present in your authorization policy which is enforces by ztunnel this will become a DENY rule as part of a safer "fail closed" security posture.
+
+This table is based on the following invariants:
+
+1. The source pod is a normal pod which has ztunnel enabled.
+1. Redirection to the waypoint is configured correctly.
+1. The waypoint is configured with the `istio.io/waypoint-for` label set to `service`.
+
+| Waypoint* | Attachment Style | Resources | Source Identity | Enforced By |
+| --- | --- | --- | --- | --- |
+| no | Selector | Pod | n/a | DENY destination ztunnel |
+| yes | Selector | Pod | n/a | DENY destination ztunnel |
+| no | _empty**_ | Namespace | n/a | DENY destination ztunnel |
+| yes | _empty**_ | Namespace | n/a | DENY destination ztunnel |
```

Comment:
Do you need `DENY` before `destination ztunnel` for above 4 rows?  could be allow :)

Reply:
Also, I don't understand why source identity is `n/a`. Wouldnt that be the client or waypoint if it exists?